04/app.js

```diff
@@ -2,8 +2,33 @@ const user = {
     firstName: 'Adam',
     lastName: 'Nowak',
     born: {
-        day: '14',
-        month: '04',
+        day: '17',
+        month: '12',
         year: '1985'
     }
-}
\ No newline at end of file
+}
+const fullDate = getCurrentlyDate();
+
+
+function getCurrentlyDate() {
+    const date = new Date();
+    return date;
+}
+
+function checkBirthsday(date, obj) {
+    const currentlyMonth = date.getMonth() + 1;
+    const currentlyDay = date.getDate();
+
+    if(currentlyDay == obj.born['day'] && currentlyMonth == obj.born['month']) {
```

Comment:
Bezpieczniej jest używać `===` zamiast `==`. Spróbuj tak przebudować, aby to działało ;)
PS. Sprawdzaj dla miesiąca `04`. Do `Date()` możesz przekazać odpowiedni czas.

Reply:
Mateuszu, 

Czy użycie parseInt tez jest dobre ?